src/pages/BlockDetail/BlockComp.tsx

```diff
@@ -151,12 +159,106 @@ export const BlockOverview: FC<{ block: State.Block }> = ({ block }) => {
     },
     {
       title: i18n.t('block.size'),
-      content: block.size ? `${block.size.toLocaleString('en')} Bytes` : '-',
+      content: block.size ? (
+        <div
+          style={{
+            display: 'flex',
+            alignItems: 'center',
+          }}
+        >
+          {`${block.size.toLocaleString('en')} Bytes`}
+          <Tooltip
+            placement="top"
+            overlayClassName={styles.comparedSizeTooltip}
+            title={
+              <>
+                {block.largestBlockInEpoch ? (
+                  <div>
+                    <div>Compared to the max size in epoch</div>
+                    <div>
+                      {localeNumberString(block.largestBlockInEpoch)} ({blockSizePercentOfLargestInEpoch}%)
+                    </div>
+                    <Progress percent={blockSizePercentOfLargestInEpoch} showInfo={false} status="success" />
```

Comment:
Color is incorrect, expected to be `#00CC9B`/`#9a2cec` and `#346DFF`, the primary color defined in https://github.com/nervosnetwork/ckb-explorer-frontend/blob/565f2d8911759538eb56257a1b3b1ef99a96b263/src/index.css#L2

`Progress` from `antd` could be replaced by the native `progress` HTML element(https://developer.mozilla.org/en-US/docs/Web/HTML/Element/progress) if the customization is infeasible.